storage.py

```diff
@@ -18,5 +18,7 @@ def remove(self):
     def set(self):
         pass
     
-    def add(self):
-        pass
\ No newline at end of file
+    def add(self, key, value):
```

Comment:
Are you sure that it is a good idea not to return anything?

Reply:
What an 'add' method should return?
Similar python `dict.update` method doesn't return anything
https://docs.python.org/3/library/stdtypes.html#dict.update